engine/storage/volume/src/main/java/org/apache/cloudstack/storage/datastore/provider/DefaultHostListener.java

```diff
@@ -166,6 +166,13 @@ private void updateStoragePoolHostVOAndDetails(StoragePool pool, long hostId, Mo
                 storagePoolDetailsDao.persist(storagePoolDetailVO);
             }
         }
```

Comment:
why on host connect again? as this is addressed / updated through API call.

Reply:
This path is used when the pool scope is zone level.